### PR TITLE
Restructure dataproc components

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/542)

### CODE REVIEW

File: .cloudbuild.yaml [hunk 1]
```diff
@@ -125,27 +125,27 @@ steps:
 # Build the Dataproc-based pipeline component images
 - name: 'gcr.io/cloud-builders/docker'
   entrypoint: '/bin/bash'
-  args: ['-c', 'cd /workspace/components/dataproc/containers/analyze && ./build.sh -p $PROJECT_ID -t $COMMIT_SHA']
+  args: ['-c', 'cd /workspace/components/dataproc/analyze/src && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
```
Comment (Ark-kun) [inline .cloudbuild.yaml]: No `/src` in these paths.

**Comment by Ark-kun** [comment]
Thanks for your effort in this restructuring!
The only one remaining will be the `resnet-cmle` group.

File: .cloudbuild.yaml (hunk 1 above)
Comment (gaoning777) [reply .cloudbuild.yaml]: you are correct. Done

**Pull request merged**